lib/pure/terminal.nim

```diff
@@ -635,15 +635,17 @@ proc ansiForegroundColorCode*(color: Color): string =
 
 template ansiForegroundColorCode*(color: static[Color]): string =
   const rgb = extractRGB(color)
-  (static(fmt"{fgPrefix}{rgb.r};{rgb.g};{rgb.b}m"))
+  # no usage of `fmt`, see issue #7632
+  (static($fgPrefix & $(rgb.r) & ";" & $(rgb.g) & ";" & $(rgb.b) & "m"))
```

Comment:
Would `%` (imported from `strutils`) work here, for a slightly better looking code? (and the same for the other snippet)

```
(static("$1$2;$3;$4m" % [$fgPrefix, $(rgb.r), $(rgb.g), $(rgb.b)]))
```

Reply:
Yeah, good point. That looks a lot nicer!